ASM/src/player_save_mask.asm

```diff
@@ -0,0 +1,6 @@
+player_save_mask:
+    lbu     t0, 0x3B(s2) ;saveContext->savedMask (custom field)
+    sb      t0, 0x014F(a1) ;player->currentMask
+
+    j       0x8038C9A0 ; Pre relocated jump. Player is always in the same place so it *should* be fine.
```

Comment:
I would avoid hard-coding addresses if possible. See #1622

Reply:
Alright I found a way to do this properly. It now no longer uses a hard coded address and properly relocates the function.